.gitlab-ci.yml

```diff
@@ -13,6 +13,16 @@ variables:
   RASPBIAN_URL: https://downloads.raspberrypi.org/raspios_lite_armhf/images/raspios_lite_armhf-2021-01-12/2021-01-11-raspios-buster-armhf-lite.zip
   RASPBIAN_NAME: 2021-01-11-raspios-buster-armhf-lite
 
+  # Which version of mender-convert to use in published image name. Normally
+  # empty, in which case the branch or tag name will be used, but this variable
+  # is available so that it's possible to make small build fixes on an already
+  # tagged version.
+  MENDER_CONVERT_PUBLISH_VERSION: ""
+
+  # Whether to publish images. Note that images from branches are always
+  # published, even if this is false.
+  PUBLISH_MENDER_CONVERT: "false"
```

Comment:
Btw, I realize this should be `PUBLISH_MENDER_CONVERT_IMAGES`. I'll change it, no need to comment on it.